layers/+themes/themes-megapack/packages.el

```diff
@@ -37,7 +37,8 @@
     dracula-theme
     espresso-theme
     farmhouse-theme
-    firebelly-theme
+    ;; Removed from melpa
+    ; firebelly-theme
```

Comment:
I did this way because this file has some other themes that are commented in the same way. One semicolon for the actual theme that was "disabled" and two semicolons for the explanation why it was disabled.

I can either do like the rest of the file (current revision), or do it as you suggested. I have no particular strong opinion on it. What do you prefer?